src/Libraries/CoreNodeModels/Logic/AndOr.cs

```diff
@@ -31,9 +31,9 @@ protected BinaryLogic(Operator op)
         {
             _op = op;
 
-            InPorts.Add(new PortModel(PortType.Input, this, new PortData("bool0", Resources.PortDataOperandToolTip)));
-            InPorts.Add(new PortModel(PortType.Input, this, new PortData("bool1", Resources.PortDataOperandToolTip)));
-            OutPorts.Add(new PortModel(PortType.Output, this, new PortData("", Resources.PortDataResultToolTip)));
+            InPorts.Add(new PortModel(PortType.Input, this, new PortData("boolean0", Resources.PortDataOperandToolTip + 0)));
```

Comment:
I think `bool` is perfectly okay given that many languages commonly use `bool` as a short form of "boolean" in favour of brevity. @Amoursol ?

Reply:
@aparajit-pratap - Our premise here is that Dynamo is a 'bridge language' between the worlds of Design and Comp-Sci. While all of these terms are common in CS, they are a lot harder to wrap your head around coming from the AEC industry. I'm in favour of the more verbose `boolean`, and ditto for `string` over `str` because it's easier to find information on when a user does research, and more consistent across the library and nodes.

Reply:
Adressed in 9e1a3c0.
Changed from boolean to bool to follow DesignScript base type abbreviations